lib/src/feature/authentication/sign_up/view/sign_up_screen.dart

```diff
@@ -1,17 +1,37 @@
+// ignore_for_file: use_build_context_synchronously
```

Comment:
NOTE: if showing a warning on some lines have `context`.You can check `if(!context.mounted) return` before that line